sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### Other Changes
 
+- Update swagger to newest version
```

Comment:
We don't need this - updating swagger doesn't affect client facing API so nothing "changed" :)